tests/client/amo/components/TestAddonDetail.js

```diff
@@ -2,16 +2,20 @@ import React from 'react';
 import { findDOMNode } from 'react-dom';
 import {
   Simulate,
+  scryRenderedComponentsWithType,
```

Comment:
I think it's supposed to be "scary." One day maybe we can switch it all to the new hotness. https://github.com/mozilla/addons-frontend/issues/1104